### ISSUE TITLE
Accessibility - Can Only Tap Down Arrow to See Content on Steps Screens

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#13]
While testing the eight Steps screens I noticed you only can open the content by tapping on the down arrow instead of the words and the down arrow.

Wondering if it is possible to make the text and the down arrow all one button to be toggled. Because as of now VoiceOver is repeating the text twice once for the text part and again as the button. It's a bit confusing to hear the same text twice in a row.

The problem applies to all the Steps screens.

**NOTE** - My testing is being done on our iPad Air 2 (iOS 10.3.3) because our iPod Touch 5th Generation (iOS 9.3.5) opens the application but displays a blank screen.

**Comment by adriennerlong** [comment]
I've flagged this as a high priority issue. We will ask the developers to address before 9/11.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Thanks, for getting them to work on that.

**Comment by jfc3-dol** [comment]
Closing this issues because it has been fixed on the screens I tested.

I still think the sections should be open so people can see and easily read the content,.
